### DISCUSSION TITLE
Consistent Alert Settings

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#3697] [Ideas]
**Current status**
Only alerts under "Glucose Level Alerts List" are fully customizable.  The feature request is about the rest of the alerts.
There is only one override silent mode setting for all the other alerts.
There is no ability to enable or disable the vibrator for these alerts.
These alerts do not follow the chosen volume profile for all alerts.
  

|                           | Glucose Level | Missed Reading | Rising/Falling fast | Persistent High | Forecast Low |
| ----------------- | --------------- | ------------------ | ------------------- | ----------------- | -------------- |
| Override Silent   |  Yes                | Yes                      | Yes                        | Yes                     | Yes                |
| Vibrate choice    | Yes                 | No                     | No                         | No                     | No                |
| Time of day        | Yes                 | Yes                     | No                         | No                     | No                |
| Tone Choice       | Yes                 | Yes                     | Yes                         | Yes                    | Yes                |
| Volume Profile   | Yes                 | No                      | No                         | No                     | No                |  

\* Inherited from `Settings` &#8722;> `Alarms and Alerts` &#8722;> `Other Alerts`  
<br/>

**Preferred Solution**
Please enhance the other alerts (calibration, missed reading, fast rising, fast falling, persistent high, and forecast low) to offer the missing features described above.

**Existing related issues**
https://github.com/NightscoutFoundation/xDrip/issues/555  

**Pull Requests**

**Comment by thedewi** [comment]
I would also like to control how often these repeat. "Forecast low" is the most useful alarm but it only repeats after 20 minutes, which is enough time for a "wait and see" to become "too late to prevent the hypo"

**Comment by Navid200** [comment]
This isn't really an issue.
It is more of a discussion.
